src/client/pages/VergunningDetail/StatusLineItems.tsx

```diff
@@ -13,13 +13,12 @@ function useVergunningStatusLineItems(vergunning?: Vergunning) {
     }
 
     const isDone = vergunning.status === 'Afgehandeld';
+    let hasWorkflow = false;
     let dateWorkflowActive = vergunning.dateRequest || '';
```

Comment:
Of moet deze gewoon standaard leeg zijn? Dan heb je de boolean niet nodig.

Reply:
Oh nee toch niet, want je kunt !dateWorkflowActive hebben maar wel een vergunningstype zijn met de mogelijkheid tot een workflow...

Reply:
@timvanoostrom

Reply:
Hmmja. Er zijn idd ook workflows waarbij de binnenkomende aanvraag direct in behandeling wordt genomen en dan is het inderdaad dateRequest die we willen tonen als in behandeling

Reply:
Of misschien moeten we de `dateWorkflowActive` gewoon overal vullen in de backend. Dan kunnen we die logica hier weghalen.

Reply:
Ja, dat lijkt me ook prima. Zal ik dat doen?

Reply:
En dan kijken naar de status 'in behandeling' ?

Reply:
Ja doemaar. Check ook ff of dateWorkflowActive bij alle Vergunningtypes is toegevoegd in de openapi.yml file